### PR TITLE
master-bl-39

### PR DESCRIPTION
**Pull request by xbrianlee**
https://issues.liferay.com/browse/LRQA-24101

Please backport this to 7.0.x.

**This will not pass poshi-validation. Please wait until jobs move over to test-1 to merge this.**

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/macros/MobileDevice.macro [hunk 1]
```diff
@@ -36,13 +38,13 @@
 	<command name="assertRedirectActionPG">
 		<execute macro-mobile="MobileNavigator#clickReorderButton" />
 
-		<execute function="AssertClick" locator1="Home#PAGE" value1="${pageName}">
-			<var name="key_pageName" value="${pageName}" />
-		</execute>
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/macros/MobileDevice.macro]: Do not nest variables within functions if possible

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/sites/mobiledevicefamilies/MobileDeviceFamilies.testcase [hunk 2]
```diff
@@ -38,10 +38,10 @@
 		</execute>
 
 		<execute macro="MobileDevice#addRuleCP">
+			<var name="deviceType" value="Any" />
 			<var name="mobileDeviceFamilyName" value="Mobile Device Family Name" />
-			<var name="ruleName" value="Rule Name" />
 			<var name="operatingSystem" value="Any Operating System" />
-			<var name="deviceType" value="Any" />
+			<var name="ruleName" value="Rule Name" />
 		</execute>
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/sites/mobiledevicefamilies/MobileDeviceFamilies.testcase]: Alphabetize vars

**Comment by alee8888** [comment]
ci:retest

**Comment by alee8888** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by xbrianlee** [comment]
Updated this PR with the fix for the navigator macro.

**Comment by alee8888** [comment]
Kiana will resend, closing.

**Pull request closed without merging**